playbooks/set-osd-global-config.yml

```diff
@@ -0,0 +1,20 @@
+- name: set osd_pool_default_size
+  hosts: rhcs5-admin
+  become: true
+  gather_facts: false
+  tasks:
+    - name: set the osd_pool_default_size configuration
+      ceph_config:
+        action: set
+        who: global
+        option: osd_pool_default_size
+        value: 2
+    - name: get the osd_pool_default_size configuration
+      ceph_config:
+        action: get
+        who: global
+        option: osd_pool_default_size
+      register: verify_osd_pool_default_size
+    - name: print current osd_pool_default_size setting
+      debug:
+          msg: "the value of 'osd_pool_default_size' is {{ verify_osd_pool_default_size.stdout }}"
```

Comment:
This would just print the message for the playbook. Where are we validating if the value is actually the same as what was set?

Adding a check with `ceph config dump` would be an appropriate test.